.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: required
```

Comment:
Re-roll from master and this file should disappear (this file isn't a change from this commit).